Core/include/Acts/Utilities/TypeList.hpp

```diff
@@ -10,15 +10,17 @@
 
 #include <type_traits>
 
-namespace Acts {
+#include <stdlib.h>
```

Comment:
Is this for `size_t`? I think it would be a better idea to use `std::size_t` and include e.g. `cstddef`.

Reply:
Ah, that was too late yesterday I had #2624 differently in mind.

Reply:
fixed.